### DISCUSSION TITLE
How to Enable Prettier

### DISCUSSION THREAD

**Discussion by StevenXL** [LazyVim/LazyVim#1519] [Q&A] [answered]
There seems to be first-class support for `prettier` in `LazyVim`. However, I am too dense to understand the documentation [here](https://www.lazyvim.org/extras/formatting/prettier). 

I have made the changes to my `lua/config/lazy.lua` file, per the documentation. However, I don't know what to do about the directions regarding `mason.nvim` and `null-ls.nvim`. 

Where should those files go? Do I use both or only one? 

This question has been "sorta" answered here: https://github.com/LazyVim/LazyVim/discussions/611, but not in full, so I hope it's OK to post again.

**Comment by dpetka2001** [comment, marked as answer]
You shouldn't do anything except adding that line to `lazy.lua`. The rest just describes what is being done under the hood by adding that line. The description is provided, because when you feel comfortable enough with how things are done in Lazyvim, you might want to tinker with the default configuration Lazyvim provides to adjust it more to your needs.

Edit: essentially it imports the spec in this [file](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/extras/formatting/prettier.lua). As you can see the website just describes what is being done in this file.

**Reply by StevenXL** [reply]
Thank you very much for such a clear and concise answer!
